src/protocols/BaseProtocolAdapter.ts

```diff
@@ -9,4 +14,30 @@ export abstract class BaseProtocolAdapter {
   static derivationPath: string;
 
   abstract getBalance(address: string): Promise<string>;
+
+  /**
+   * Check if the network has seen this account
+   * @param address Account address
+   */
+  abstract isAccountUsed(address: string): Promise<boolean>;
+
+    /**
+   * get address from the keys
+   * @param secretKey Secret key
+   * @param chainCode Chaincode
```

Comment:
This lines gives no value in my opinion.

Reply:
Removed it.